src/QMCDrivers/DMC/DMCDriverInput.cpp

```diff
@@ -43,6 +43,9 @@ void DMCDriverInput::readXML(xmlNodePtr node)
     throw std::runtime_error("Illegal input for MaxAge in DMC input section");
   if(branch_interval_ < 0)
     throw std::runtime_error("Illegal input for branchInterval or substeps in DMC input section");
+
+  if(reconfig_str != "no" && reconfig_str != "runwhileincorrect")
+    APP_ABORT("Reconfiguration is currently broken and gives incorrect results. Set reconfiguration=\"no\" or remove the reconfiguration option from the DMC input section. To run performance tests, please set reconfiguration to \"runwhileincorrect\" instead of \"yes\" to restore consistent behaviour.")
 }
```

Comment:
leaving aside that touching the reconfig_str after its been read breaks the new input model, i.e. options are parsed once and if they are typed data they are that type not a string converted in situ all over the code.

A valid value for reconfig_str is empty.